packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -4,18 +4,25 @@ import { feature } from '../lib/featureManagement';
 import { FeedLayout } from '../lib/featureValues';
 import { SharedFeedPage } from '../components/utilities';
 import { ActiveFeedContext } from '../contexts';
+import { AllFeedPages } from '../lib/query';
+
+interface UseFeedLayoutProps {
+  feedName?: AllFeedPages;
+}
 
 interface UseFeedLayout {
   shouldUseFeedLayoutV1: boolean;
 }
 
-export const useFeedLayout = (): UseFeedLayout => {
+export const useFeedLayout = ({
+  feedName: feedNameProp,
```

Comment:
These shouldn't be added based on the comment above about feedName being taken from context.

Reply:
Unfortunately the Context starts from the components after the `Feed`. So from where we called it, it is undefined.

Reply:
In that case we better of moving the context perhaps?
Which area is not under it, basically all feeds should be.

This feels like a very weird fix now, either it's automagical or we pass it.

Reply:
Part of the requirements of the context is the `items` object, moving it would involve refactoring a lot of the logic and drilling at least 6 props to this Feed component. Not sure if would be beneficial to do that amount of work and changes for this experiment.

Reply:
I see indeed, that's a bit sad.
Anyway maybe with leanfeed we can refactor it a bit :)

Reply:
Yes, agree with that. Hopefully we can introduce a context when the cards/list items can utilize to avoid all the drillings.